locale.go

```diff
@@ -10,7 +10,8 @@ import (
 // DefaultLocaleCode is the default locale used by Goment if not set.
 const DefaultLocaleCode = "en"
 
-var supportedLocales = map[string]locales.LocaleDetails{
+// SupportedLocales exports the list of available locales.
+var SupportedLocales = map[string]locales.LocaleDetails{
```

Comment:
This could potentially allow someone to modify this map from outside, right? Maybe it would be better to leave this private, and expose a public method that just returns a copy of the map, or a map of just the keys?